.circleci/config.yml

```diff
@@ -173,6 +175,16 @@ jobs:
           name: Verify locales
           command: yarn verify-locales --quiet
 
+  test-lint-shellcheck:
+    docker:
+      - image: circleci/node:10.16-browsers
```

Comment:
One small nitpick, sorry! I snuck #7447 in underneath you, so can we bump this here too:

```suggestion
      - image: circleci/node:10.17-browsers
```

Reply:
For "consistency" even though it doesn't use Node 😅